src/Symfony/Component/Validator/Constraints/ControllerArgument.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+/**
+ * @author Dynèsh Hassanaly <artyum@protonmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
```

Comment:
Compared to the other constraints, I didn't add the following part:
```php
/**
 * @Annotation
 * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
 */
```

Because this constraint is only used on a method parameter, so I only added the attribute that allows that (`#[\Attribute(\Attribute::TARGET_PARAMETER)]`). Let me know if it's correct.